docs/sources/configure-server/configure-object-storage-backend.md

```diff
@@ -12,6 +12,8 @@ aliases:
 Pyroscope can use different object storage services to persist blocks containing the profiles data.
 Blocks are flushed by ingesters [on disk]({{<relref "./configure-disk-storage.md">}}) first then are uploaded to the object store.
 
+At the moment, querying from object storage only works in microservices mode and for labels being actively ingested. In the future, we plan to support querying from object storage in standalone mode as well.
+
```

Comment:
As per slack the stand-alone mode has support for querying from object store.

The limitation for labels still exists for both stand-alone & micro-services. But once #2413 merges it also is gone.

Reply:
yeah will just close this then